.github/workflows/create-release.yml

```diff
@@ -42,6 +48,7 @@ jobs:
         pip freeze | grep -v "-e git+" | grep -v "torch" | grep -v "certifi @ file" > requirements-freeze.txt
         conda deactivate
         grep "# append_to_freeze" requirements.txt >> requirements-freeze.txt
+        grep "torch" requirements.txt >> requirements-freeze.txt
```

Comment:
As it is, I think this solution would copy several more lines than intended; specifically, lines 12, 13, 14, 16 from this snippet:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/7a62febc30a5b3abb376e8a64ab543dfbf6fade2/requirements.txt#L12-L17

In this case it happens to be fairly harmless lines (some comments, and an `--extra-index-url` that already has `# append_to_freeze`), but I don't like the potential for accidents in the future.

Why not just use the existing `# append_to_freeze` mechanism for the specific line(s) we want to keep?

Reply:
...that is a much simpler solution. :facepalm: 

I think I got flustered enough from the tests failing that I didn't see the forest for the trees here. :)

Reply:
That's ok! It's why we have reviews :heart: